lib/internal/error-serdes.js

```diff
@@ -14,9 +20,9 @@ const GetOwnPropertyNames = Object.getOwnPropertyNames;
 const DefineProperty = Object.defineProperty;
 const Assign = Object.assign;
 const ObjectPrototypeToString =
-    Function.prototype.call.bind(Object.prototype.toString);
-const ForEach = Function.prototype.call.bind(Array.prototype.forEach);
-const Call = Function.prototype.call.bind(Function.prototype.call);
+    FunctionPrototype.call.bind(ObjectPrototype.toString);
```

Comment:
FunctionPrototype appears to be coming from `primordials`, but `.call` is still a Function.prototype based value it seems?

Reply:
What do you mean by `Function.prototype based value`? This should be equivalent to

```
// Prior to bootstrap
const FunctionPrototype = Object.create(null);
Reflet.defineOwnProperty(FunctionPrototype, 'call', Reflect.getOwnPropertyDescriptor(Function.prototype, 'call'));

// At the point error-serdes.js is executed
// FunctionPrototype.call.bind(ObjectPrototype.toString);
```

Reply:
I think brad means the `.bind`

Reply:
Yes, the copyProps operation only makes 1 level of safe lookups, but `.call.bind` is 2 levels

Reply:
This is already the case in the previous code, so I don't think this PR has to address the double lookup - I could also revert the lines here and restore them back to `global.Function.call.bind`, and let future contributors migrate calls like this to `FunctionPrototype.bind` incrementally.

Reply:
this is fine, we can migrate these later